ios/Pods/MobileCenter/MobileCenter-SDK-iOS/MobileCenter.framework/Headers/MSConstants.h

```diff
@@ -0,0 +1,47 @@
+#import <Foundation/Foundation.h>
```

Comment:
Alrighty, I certainly don't want to debate the architecture you've chosen here. I'll circle in with the team and see how we feel about adding this in.